text/0002-searchable-display-attributes-behaviors.md

```diff
@@ -0,0 +1,123 @@
+- Title: Searchable Attributes and Displayed Attributes behaviors
+- Start Date: 2020-11-02
+- specification PR: meilisearch/specifications#3
+- Meilisearch Issue: 
+
+# `searchable_attributes` & `displayed_attributes` behaviors
+
+## First section: Feature Description and Interaction
+
+This first part has a general audience. It should be as little technical as possible (think user-level). This section contains 4 sub-sections:
+
+### Summary
+
+Ever since the beginning, the design of `searchableAttributes` and `displayedAttributes` as shown flaws. The introduction of the wildcard tried to adress some of those flaws, but there are still issues that this specs tries to correct.
```

Comment:
```suggestion
Ever since the beginning of MeiliSearch, the design of `searchableAttributes` and `displayedAttributes` has shown flaws. The introduction of the wildcard tried to address some of those flaws, but there are still issues that this spec tries to correct.
```